conf.d/redisdb.yaml.example

```diff
@@ -30,3 +30,11 @@ instances:
     # Default: True
     #
     # warn_on_missing_keys: True
+
+    # Max number of entries to fetch from the sloq query log
+    # By default, the check wil read this value from the redis config
+    # But if it's too high, it will default to 128
```

Comment:
"too high" should be spelled out, why is too high too high? "Higher than 128, due to potential increased latency to retrieve more than 128 slowlog entries every 15 seconds"...